src/indexers/indexer.utils.ts

```diff
@@ -84,7 +84,7 @@ export async function constructVaultDefinition(
 
   try {
     const existingDefinition = await chain.vaults.getVault(vault.address);
-    if (existingDefinition.lastHarvestIndexedBlock > createdAtBlock) {
+    if (existingDefinition.lastHarvestIndexedBlock && existingDefinition.lastHarvestIndexedBlock > createdAtBlock) {
```

Comment:
this can cause issues since older defs didn't have the block and so resulting in `NaN`